From f7e50da97db642ae8927cb171144543ecf44a48d Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:07:02 -0400
Subject: [ethernet] qlcnic: Remove duplicate object file from Makefile

Message-id: <1408626422-3941-139-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88667
O-Subject: [PATCH RHEL 7.1 138/138] net: ethernet: qlogic: qlcnic: Remove duplicate object file from Makefile
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Andreas Ruprecht <rupran@einserver.de>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id 7a6cdb8cfebf16907926e78a30f493fa587695e6

In the Makefile, qlcnic_minidump.o is included twice in the list of
object files linked into qlcnic.o.

This change removes the superfluous include.

Signed-off-by: Andreas Ruprecht <rupran@einserver.de>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/Makefile b/drivers/net/ethernet/qlogic/qlcnic/Makefile
index a848d29..3c2c2c7 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/Makefile
+++ b/drivers/net/ethernet/qlogic/qlcnic/Makefile
@@ -8,7 +8,7 @@ qlcnic-y := qlcnic_hw.o qlcnic_main.o qlcnic_init.o \
  qlcnic_ethtool.o qlcnic_ctx.o qlcnic_io.o \
  qlcnic_sysfs.o qlcnic_minidump.o qlcnic_83xx_hw.o \
  qlcnic_83xx_init.o qlcnic_83xx_vnic.o \
- qlcnic_minidump.o qlcnic_sriov_common.o
+ qlcnic_sriov_common.o
 
 qlcnic-$(CONFIG_QLCNIC_SRIOV) += qlcnic_sriov_pf.o
 
-- 
1.7.1